src/lib/Scenes/Map/GlobalMap.tsx

```diff
@@ -72,6 +85,19 @@ enum DrawerPosition {
 }
 
 export class GlobalMap extends React.Component<Props, State> {
+  /** Makes sure we're consistently using { lat, lng } internally */
+  static coordArrayToLocation(arr: [number, number] | undefined) {
```

Comment:
Love the move towards being more explicit about this **eternal** problem 💯 

As long as you're doing so what about changing this to `lngLatArrayToLocation` since otherwise it's not _immediately_ clear what the order of the array is - `[lat, lng]` vs `[lng, lat]`.

Reply:
Good idea, done'd